.gitignore

```diff
@@ -13,3 +13,4 @@ media/
 *.iml
 *.kate-swp
 data/
+venv
```

Comment:
File missing last newline. (You might want to check if your editor supports auto-adding it?)